students/rob_sanchez/Lesson 2/Fizz_Buzz.py

```diff
@@ -0,0 +1,16 @@
+def FizzBuzz():
```

Comment:
The convention in Python is to use `snake_case` for functions and variables and `CamelCase` for class names